extra-multimedia/pipewire/autobuild/defines

```diff
@@ -18,6 +18,14 @@ BUILDDEP="avahi doxygen docutils graphviz meson ninja pulseaudio sdl2"
 ABTYPE=meson
 MESON_AFTER="-Ddocs=enabled \
              -Dman=enabled \
+             -Dgstreamer=enabled \
+             -Dsystemd=enabled \
+             -Dgstreamer-device-provider=disabled
+             -Dsdl2=disabled \
+             -Daudiotestsrc=disabled \
+             -Dvideotestsrc=disabled \
+             -Dvolume=disabled \
+             -Dbluez5-codec-aptx=enabled \
```

Comment:
Please disable JACK support explicitly to prevent default changes down the road.

Reply:
Done.